use_sso_authentication/index.rst

```diff
@@ -0,0 +1,106 @@
+Use SSO authentication locally
+==============================
+
+This chapter describes how to enable the SSO authentication for development purpose.
+
+How SSO works ?
+---------------
+
+| SSO means `Single Sign-On`, their is different implementation/protocols in order to implement a SSO authentication mecanism.
+| We implemented the ``SAMLv2`` protocol as it becomes a standard in the industry.
+|
+
+    Security Assertion Markup Language 2.0 (SAML 2.0) is a version of the SAML standard for exchanging authentication and authorization data between security domains.
+    SAML 2.0 is an XML-based protocol that uses security tokens containing assertions to pass information about a principal (usually an end user) between a SAML authority, named an ``Identity Provider``, and a SAML consumer, named a ``Service Provider``.
+
+    -- Source `Wikipedia <https://en.wikipedia.org/wiki/SAML_2.0>`_
+
+The ``Identity Provider`` offer user authentication as a service, and the ``Service Provider`` could be considered as a client that will ask if a user is authenticated.
+
+The content exchanged between the IdP, and the SP must be sign, that why we have to configure thoses entity with ``Certificates`` and ``Private keys``.
+
+All the exchanges, redirections between the browser, the Service Provider and the Identity Provider could be represented with the schema below
+
+.. image:: https://upload.wikimedia.org/wikipedia/en/0/04/Saml2-browser-sso-redirect-post.png
+
+
+Setting up the Identity Provider (IdP)
+--------------------------------------
+
+Using Docker
+^^^^^^^^^^^^
+
+We provide an already configured IdP server image in the ``docker-compose.yml`` file, named ``sso-idp-server``.
+
+The following information are here if you want to tweak something, if not, go directly to the `Setting up the Service Provider (SP)` section.
+
+
+This container will expose the ``8081`` port, you can override this value in the ``.env`` file.
+
+You can access the IDP administration page at the following url: http://localhost:8081/simplesaml .
+
+| Some users are already configured, the same that are in the `icecat_demo_dev fixtures <https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/users.csv>`_.
```

Comment:
Not sure it's a good idea to point to the master branch :/

Reply:
yep, have to be updated when the 3.0 will be created.